HHAuto.user.js

```diff
@@ -3378,7 +3389,7 @@ var  CrushThem = function()
                 if (league_battle !== null && league_battle === "1")
                 {
                     logHHAuto("Reloading after league fight.");
-                    setTimeout(function(){location.reload();},randomInterval(3000,5000));
```

Comment:
linked to #222

Reply:
Did it solve the issue ?
That is a long timeout ! That's not a problem for league, but for troll fights, especially with the mythic, I think it is too long. 
We'll have to find a way to better manage server lag.

Reply:
@YotoTheOne, this one only do for league, for troll it is going to home (to check back event)

Reply:
@Roukys : yes I know ;) 
I said that because you set the timeout after the troll fight to 2000-4000, which is also pretty high, and I think it was for the same reason (#222 ). Sorry I wasn't clear enough ^^